sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/HoldOptions.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.models;
+
+import com.azure.communication.common.CommunicationIdentifier;
+
+/**
+ * Options for the Hold operation.
+ */
+public class HoldOptions {
```

Comment:
this is missing the optional call back URI.

Reply:
added optional call back URI